data/abi/abi_type.go

```diff
@@ -333,8 +333,8 @@ func makeDynamicArrayType(argumentType Type) Type {
 	}
 }
 
-// makeTupleType makes tuple ABI type by taking an array of tuple element types as argument.
-func makeTupleType(argumentTypes []Type) (Type, error) {
+// MakeTupleType makes tuple ABI type by taking an array of tuple element types as argument.
+func MakeTupleType(argumentTypes []Type) (Type, error) {
 	if len(argumentTypes) >= math.MaxUint16 {
 		return Type{}, fmt.Errorf("tuple type child type number larger than maximum uint16 error")
 	}
```

Comment:
I kinda want to get this exported. It seems @algoidurovic is working on https://github.com/algorand/go-algorand-sdk/pull/258/ and he needs this stuff?

Reply:
Right, this makes sense